index.js

```diff
@@ -1,4 +1,4 @@
-const zeroInCelcius = 32 // This is the Celcius equivalent to the freezing point
+const zeroInCelcius = 31 // This is the Celcius equivalent to the freezing point
```

Comment:
The zeroInCelicius value is 32, please fix this before approval

Reply:
Fixed by resetting this constant back to 32 degrees Fahrenheit. Thanks!